### PR TITLE
Fix kyber oasis

### PR DESCRIPTION
**Pull request by g1nt0ki**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/fountain.js exports TVL: 
        
 
 
    aurora                    3.03 M
    aurora-borrowed           2.43 M
    borrowed                  2.43 M
    
    total                    3.03 M

**Comment by llamatester** [comment]
The adapter at projects/kyber/index.js exports TVL: 
        
 
 
    oasis                     4.369798089
    avalanche                 62.85 M
    ethereum                  44.93 M
    arbitrum                  44.17 k
    aurora                    0.0000010658839317121727
    bsc                       5.14 M
    cronos                    90.81 k
    polygon                   20.87 M
    fantom                    150.77 k
    
    total                    134.07 M

File: projects/fountain.js [hunk 1]
```diff
@@ -1,15 +1,6 @@
...
-        '0x3223f17957ba502cbe71401d55a0db26e5f7c68f':'0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
-        '0xdc19a122e268128b5ee20366299fc7b5b199c8e3':'0xdac17f958d2ee523a2206206994597c13d831ec7',
-        '0xe8a638b3b7565ee7c5eb9755e58552afc87b94dd': '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
-    }
-    return map[addr.toLowerCase()] || `oasis:${addr}`
-
-}
 module.exports = {
     timetravel: true,
     aurora: {
```
Comment (g1nt0ki) [inline projects/fountain.js:6]: wait, should this chain be changed to oasis?

Comment (waynebruce0x) [reply projects/fountain.js:6]: It looks like this adapter code isnt actually used on our front end

**Pull request merged**